print-mptcp.c

```diff
@@ -182,6 +183,31 @@ struct mp_prio {
 
 #define MP_PRIO_B                       0x01
 
+static const struct tok mp_tcprst_flags[] = {
+        { 0x08, "U" },
+        { 0x04, "V" },
+        { 0x02, "W" },
+        { 0x01, "T" },
+        { 0, NULL }
+};
+
+static const struct tok mp_tcprst_reason[] = {
```

Comment:
(very little) detail if you have to update this commit: just to uniform with what is above, maybe best to add `s` (`reasons`) and use `0x0X` (`0x06` instead of `0x6` like in the RFC :)

Reply:
sure, will do. Thanks for reviewing!